data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -134,12 +132,10 @@ then
 			--config-output-file=/assets/etcd-bootstrap/config \
 			--cluster-config-file=/assets/manifests/cluster-network-02-config.yml
 
-		sed -i "s/__BOOTSTRAP_IP__/${BOOTSTRAP_IP}/" /opt/openshift/manifests/etcd-host-service-endpoints.yaml
```

Comment:
lack of this is probably creating an invalid manifest, but I haven't checked.

Reply:
yeah possible, looking

Reply:
[subsets[0].addresses[0].ip: Invalid value: "__BOOTSTRAP_IP__": must be a valid IP address